app/controllers/challenges_controller.rb

```diff
@@ -54,5 +54,12 @@ def load_challenge
     @challenge = @collection.challenge
     no_challenge and return unless @challenge
   end
+
+  # When Challenges are deleted, there are two references left behind that need to be reset to nil
+  def clear_challenge_references
```

Comment:
Just wondering: would this work as an `after_destroy` or `before_destroy` callback in the prompt meme and gift exchange models rather than in the controllers?